lib/recognizer_web/controllers/fallback_controller.ex

```diff
@@ -18,7 +18,16 @@ defmodule RecognizerWeb.FallbackController do
   end
 
   @impl Guardian.Plug.ErrorHandler
-  def auth_error(conn, {_type, _reason}, _) do
+  def auth_error(conn, {:invalid_token, _reason}, _) do
+    conn
+    |> Recognizer.Guardian.Plug.sign_out()
+    |> fetch_session()
+    |> clear_session()
+    |> respond(:unauthorized, "401-invalid")
```

Comment:
Why don't we redirect to login with a flash?

Reply:
Let me go back a couple of commits to trigger a login loop and test this. I don't think that would work, but I can't really describe why I think that, so might as well try it out. Give me a bit.

Reply:
Updated. Should work because we store the get params of the url.